pkgs/helpers/regexHandler.go

```diff
@@ -5,26 +5,30 @@ import (
 	"regexp"
 )
 
+// Matches received index and returns the string
 func GetSubString(s string, index []int) string {
 	if index == nil {
 		return "empty"
 	}
 	return s[index[0]:index[1]]
```

Comment:
There is a possibility of index out of range errors. So its best to check for the length of `index` before you try to access its  values